### ISSUE TITLE
Error: Get Certificate Fail

### ISSUE THREAD

**Issue by ghost** [signpe/signpe#1]
Hi,

The tool seems to fail for a self signed (test) cert.

Github repo is empty... can you assist?

And on top of that, your contact form does not work.  :confused:  :disappointed: :astonished:

![sshot-1](https://cloud.githubusercontent.com/assets/8763290/16101660/d2fcca9a-335f-11e6-9053-af979ae20bfc.png)

**Comment by haephrati** [comment]
Thanks for your comments.
1. I never tried a tests cert. Don't know about these types of certs. We use certificates we buy and that was the purpose of this tool. I will certainly look into it.
2. The repo is empty because right now we haven't released a source code version of this product.
3. The contact form seems to have a problem but this entire site will be moved to www.securedglobe.com where you can also use the contact form in the future.

**Comment by ghost** [comment]
A test cert, meaning a self signed cert, created with openSSL

**Comment by haephrati** [comment]
Yes I know about the but technically speaking, if sign.pe doesn't work, it probably requires special handling. I use the formal Windows API for code signing so that's the only explanation I can think of.

**Comment by ghost** [comment]
kSign worked perfect for me.

SignPE looks promising but I'm unable to get it to work.

**Comment by haephrati** [comment]
Did you try real certificates?
By the way, I didn't like kSign at all. Also we WILL listen to any comment and suggestion and if you confirm the only issue is the test certificate, we will fix that and let you know in a day or two.

**Comment by ghost** [comment]
Here are the exact steps to reproduce my issue.
1. Create a simple batch (.bat) file and save it.
2. Convert it with the Bat to Exe converter from http://www.f2ko.de/en/b2e.php
3. Use the latest version of OpenSSL and follow [these](https://blog.didierstevens.com/2015/03/30/howto-make-your-own-cert-with-openssl-on-windows/) instructions.
4. Try signing the exe you created earlier.

Thanks

**Comment by haephrati** [comment]
I will get back to you
